weasyl/submission.py

```diff
@@ -1072,7 +1077,7 @@ def select_recently_popular():
             LEFT JOIN favorite ON favorite.type = 's' AND submission.submitid = favorite.targetid
         WHERE
             submission.unixtime > EXTRACT(EPOCH FROM now() - %(max_days)s * INTERVAL '1 day')::INTEGER AND
```

Comment:
Yes, it could be; thanks! However, it seems this isn’t necessary at all; PostgreSQL can do the join properly now. I don’t know if this is because we ran an `ANALYZE` on the favorite table, or because SQLAlchemy’s generated query was just that bad.